qa/setup_packages.py

```diff
@@ -439,50 +463,46 @@ def get_pyvers_name(self, url, cuda_version):
 
 
 all_packages = [PlainPackage("numpy", [">=1.17,<1.24"]),
-                PlainPackage("opencv-python", [PckgVer("4.5.1.48", dependencies=["numpy<1.24"])]),
+                PlainPackage("opencv-python", [PckgVer("4.5.4.60", dependencies=["numpy<1.24"])]),
                 CudaPackage("cupy",
-                            {"100": [PckgVer("8.6.0", dependencies=["numpy<1.24"])],
-                             "110": [PckgVer("8.6.0", dependencies=["numpy<1.24"])],
-                             "111": [PckgVer("8.6.0", dependencies=["numpy<1.24"])]},
+                            {"113": [PckgVer("9.6.0", python_max_ver="3.6",
+                                             dependencies=["numpy<1.24"]),
+                                     PckgVer("10.0.0", python_min_ver="3.7",
+                                             dependencies=["numpy<1.24"])]},
                             "cupy-cuda{cuda_v}"),
                 CudaPackage("mxnet",
-                            {"100": [PckgVer("1.9.0", dependencies=["numpy<1.24"])],
-                             "110": [PckgVer("1.9.1", dependencies=["numpy<1.24"])],
-                             "111": [PckgVer("1.9.1", dependencies=["numpy<1.24"])]},
+                            {"113": [PckgVer("1.9.1", dependencies=["numpy<1.24"])]},
```

Comment:
What is special about 113? Is this some special build with that cuda version being the highest supported?

Reply:
In case of PyTorch it seems to be the only version (with that cuda) that supports Python 3.6 and 3.10 at the same time, so we can have only one CUDA version in the test image. For other CUDA version there is either no variant with 3.6 or 3.10.

Reply:
Oh I see, thanks.